src/diffusers/pipelines/alt_diffusion/modeling_roberta_series.py

```diff
@@ -95,14 +99,21 @@ def forward(
             encoder_hidden_states=encoder_hidden_states,
             encoder_attention_mask=encoder_attention_mask,
             output_attentions=output_attentions,
-            output_hidden_states=output_hidden_states,
+            # output_hidden_states=output_hidden_states,
+            output_hidden_states=True,
```

Comment:
Only necessary if we're in m18 mode - we still need to support the other use case